frontend/src/components/Seats.vue

```diff
@@ -0,0 +1,143 @@
+<script setup>
+import { ref, onMounted } from "vue";
+import { defineProps } from 'vue';
+
+const props = defineProps({
+  movie: {
+    type: Object,
+    required: true
+  }
+});
+
+const cinemaHall = {
+  name: "Hall 1",
+  seatCount: 56
+};
+
+const schedule = {
+  id: "67156dab33fe431712f1cbf3",
+  localDateTime: "2024-10-15T10:18:00",
+  cinemaHall: cinemaHall, 
+  movie: props.movie  
+};
+
+const user = {
```

Comment:
oops! can we have a way to store senetive data in the frontend aswell as in backend?